libs/api_common/api_common.hpp

```diff
@@ -5,7 +5,18 @@
 #include "ebpf_api.h"
 #include "ebpf_execution_context.h"
 #include "ebpf_utilities.h"
+
+#pragma warning(push)
+#pragma warning(disable : 26439) // This kind of function should not throw. Declare it 'noexcept'.
```

Comment:
question: are these warnings disabled due to verifier code that is external?

Reply:
Correct, platform.hpp is in the verifier repo